Two tests in test_21_Circuit are failing

### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

These two tests are failing on main as of d1830e889934269ff70b0c043b2853ada3b4b84c:

```
(venv) PS C:\git\Ansys\pyaedt> pytest _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation
=================================================================================== test session starts ====================================================================================
platform win32 -- Python 3.9.12, pytest-7.2.1, pluggy-1.0.0
rootdir: C:\git\Ansys\pyaedt, configfile: pytest.ini
plugins: cov-4.0.0, xdist-3.2.0
collected 2 items

_unittest\test_21_Circuit.py FF                                                                                                                                                       [100%]

========================================================================================= FAILURES ========================================================================================= 
________________________________________________________________________ TestClass.test_06b_add_3dlayout_component _________________________________________________________________________ 

self = <_unittest.test_21_Circuit.TestClass object at 0x0000019DA0EE0C10>

    def test_06b_add_3dlayout_component(self):
        myedb = self.aedtapp.modeler.schematic.add_subcircuit_3dlayout("Galileo_G87173_204")
        assert type(myedb.id) is int
        ports = myedb.pins
        tx = ports
        rx = ports
        insertions = ["dB(S({},{}))".format(i.name, j.name) for i, j in zip(tx, rx)]
>       assert self.aedtapp.post.create_report(
            insertions,
            self.aedtapp.nominal_adaptive,
            plotname="Insertion Losses",
            plot_type="Rectangular Plot",
            report_category="Standard",
            subdesign_id=myedb.id,
        )
E       AssertionError: assert False
E        +  where False = <bound method PostProcessorCommon.create_report of <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000019DC0F9BD90>>(['dB(S(J3B2.3.USBH2_DP_CH,J3B2.3.USBH2_DP_CH))', 'dB(S(J3B2.2.USBH2_DN_CH,J3B2.2.USBH2_DN_CH))', 'dB(S(J2B3-2,J2B3-2))'], '', plotname='Insertion Losses', plot_type='Rectangular Plot', report_category='Standard', subdesign_id=60)
E        +    where <bound method PostProcessorCommon.create_report of <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000019DC0F9BD90>> = <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000019DC0F9BD90>.create_report
E        +      where <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000019DC0F9BD90> = <pyaedt.circuit.Circuit object at 0x0000019D9F253B20>.post
E        +        where <pyaedt.circuit.Circuit object at 0x0000019D9F253B20> = <_unittest.test_21_Circuit.TestClass object at 0x0000019DA0EE0C10>.aedtapp
E        +    and   '' = <pyaedt.circuit.Circuit object at 0x0000019D9F253B20>.nominal_adaptive
E        +      where <pyaedt.circuit.Circuit object at 0x0000019D9F253B20> = <_unittest.test_21_Circuit.TestClass object at 0x0000019DA0EE0C10>.aedtapp
E        +    and   60 = <pyaedt.modeler.circuits.object3dcircuit.CircuitComponent object at 0x0000019DC0D77D30>.id

_unittest\test_21_Circuit.py:107: AssertionError
---------------------------------------------------------------------------------- Captured stdout setup ----------------------------------------------------------------------------------- 
pyaedt INFO: using existing logger.
pyaedt INFO: Launching PyAEDT outside AEDT with CPython and PythonNET.
pyaedt INFO: AEDT installation Path C:\Program Files\AnsysEM\v221\Win64.
pyaedt INFO: Launching AEDT with module PythonNET.
pyaedt INFO: Ansoft.ElectronicsDesktop.2022.1 Started with process ID 18428.
pyaedt INFO: pyaedt v0.7.dev0
pyaedt INFO: Python version 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)]
pyaedt INFO: AEDT file load (threaded) time: 0.17030119895935059
pyaedt INFO: Project Galileo_t21 has been opened.
pyaedt INFO: No consistent unique design is present. Inserting a new design.
pyaedt INFO: Added design 'Circuit Design_WLV' of type Circuit Design.
pyaedt INFO: Aedt Objects initialized
pyaedt INFO: Variable Manager initialized
pyaedt INFO: Design Loaded
pyaedt INFO: Successfully loaded project materials !
pyaedt INFO: Materials Loaded
pyaedt INFO: AEDT file load (threaded) time: 0.04727625846862793
pyaedt INFO: Project differential_pairs has been opened.
pyaedt INFO: Active Design set to Circuit1
pyaedt INFO: Aedt Objects initialized
pyaedt INFO: Variable Manager initialized
pyaedt INFO: Design Loaded
pyaedt INFO: Successfully loaded project materials !
pyaedt INFO: Materials Loaded
------------------------------------------------------------------------------------ Captured log setup ------------------------------------------------------------------------------------
INFO     Global:aedt_logger.py:473 using existing logger.
INFO     Global:aedt_logger.py:473 Launching PyAEDT outside AEDT with CPython and PythonNET.
INFO     Global:aedt_logger.py:473 AEDT installation Path C:\Program Files\AnsysEM\v221\Win64.
INFO     Global:aedt_logger.py:473 Launching AEDT with module PythonNET.
INFO     Global:aedt_logger.py:473 Ansoft.ElectronicsDesktop.2022.1 Started with process ID 18428.
INFO     Global:aedt_logger.py:473 pyaedt v0.7.dev0
INFO     Global:aedt_logger.py:473 Python version 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)]
INFO     Global:aedt_logger.py:473 AEDT file load (threaded) time: 0.17030119895935059
INFO     Global:aedt_logger.py:473 Project Galileo_t21 has been opened.
INFO     Global:aedt_logger.py:473 No consistent unique design is present. Inserting a new design.
INFO     Global:aedt_logger.py:473 Added design 'Circuit Design_WLV' of type Circuit Design.
INFO     Global:aedt_logger.py:473 Aedt Objects initialized
INFO     Global:aedt_logger.py:473 Variable Manager initialized
INFO     Global:aedt_logger.py:473 Design Loaded
INFO     Global:aedt_logger.py:473 Successfully loaded project materials !
INFO     Global:aedt_logger.py:473 Materials Loaded
INFO     Global:aedt_logger.py:473 AEDT file load (threaded) time: 0.04727625846862793
INFO     Global:aedt_logger.py:473 Project differential_pairs has been opened.
INFO     Global:aedt_logger.py:473 Active Design set to Circuit1
INFO     Global:aedt_logger.py:473 Aedt Objects initialized
INFO     Global:aedt_logger.py:473 Variable Manager initialized
INFO     Global:aedt_logger.py:473 Design Loaded
INFO     Global:aedt_logger.py:473 Successfully loaded project materials !
INFO     Global:aedt_logger.py:473 Materials Loaded
----------------------------------------------------------------------------------- Captured stdout call ----------------------------------------------------------------------------------- 
pyaedt ERROR: Setup doesn't exist in this design.
------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------- 
ERROR    Global:aedt_logger.py:477 Setup doesn't exist in this design.
____________________________________________________________________________ TestClass.test_07a_push_excitation ____________________________________________________________________________ 

self = <_unittest.test_21_Circuit.TestClass object at 0x0000019DA0EE0D60>

    def test_07a_push_excitation(self):
>       assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=False)

_unittest\test_21_Circuit.py:126:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  
pyaedt\generic\general_methods.py:296: in wrapper
    result = user_function(*args, **kwargs)
pyaedt\circuit.py:1066: in push_excitations
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

self = <COMObject SetActiveEditor>, refDes = 'U1', options = ['NAME:options', 'CalcThevenin:=', False, 'Sol:=', 'LNA']

>   ???
E   pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147467259), None)

<COMObject SetActiveEditor>:2: com_error
pyaedt INFO: Project files removed from handlers.
---------------------------------------------------------------------------------- Captured log teardown ----------------------------------------------------------------------------------- 
================================================================================= short test summary info ================================================================================== 
FAILED _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component - AssertionError: assert False
FAILED _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation - pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147467259), None)
==================================================================================== 2 failed in 15.47s ==================================================================================== 
(venv) PS C:\git\Ansys\pyaedt> git status
On branch bug-2440
Your branch is up to date with 'origin/bug-2440'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        test_hfss3dlayout_load.py

nothing added to commit but untracked files present (use "git add" to track)
(venv) PS C:\git\Ansys\pyaedt> git checkout main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(venv) PS C:\git\Ansys\pyaedt> pytest _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation
=================================================================================== test session starts ====================================================================================
platform win32 -- Python 3.9.12, pytest-7.2.1, pluggy-1.0.0
rootdir: C:\git\Ansys\pyaedt, configfile: pytest.ini
plugins: cov-4.0.0, xdist-3.2.0
collected 2 items

_unittest\test_21_Circuit.py FF                                                                                                                                                       [100%]

========================================================================================= FAILURES ========================================================================================= 
________________________________________________________________________ TestClass.test_06b_add_3dlayout_component _________________________________________________________________________ 

self = <_unittest.test_21_Circuit.TestClass object at 0x0000021853E4EB20>

    def test_06b_add_3dlayout_component(self):
        myedb = self.aedtapp.modeler.schematic.add_subcircuit_3dlayout("Galileo_G87173_204")
        assert type(myedb.id) is int
        ports = myedb.pins
        tx = ports
        rx = ports
        insertions = ["dB(S({},{}))".format(i.name, j.name) for i, j in zip(tx, rx)]
>       assert self.aedtapp.post.create_report(
            insertions,
            self.aedtapp.nominal_adaptive,
            plotname="Insertion Losses",
            plot_type="Rectangular Plot",
            report_category="Standard",
            subdesign_id=myedb.id,
        )
E       AssertionError: assert False
E        +  where False = <bound method PostProcessorCommon.create_report of <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000021873D4E9A0>>(['dB(S(J3B2.3.USBH2_DP_CH,J3B2.3.USBH2_DP_CH))', 'dB(S(J3B2.2.USBH2_DN_CH,J3B2.2.USBH2_DN_CH))', 'dB(S(J2B3-2,J2B3-2))'], '', plotname='Insertion Losses', plot_type='Rectangular Plot', report_category='Standard', subdesign_id=60)
E        +    where <bound method PostProcessorCommon.create_report of <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000021873D4E9A0>> = <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000021873D4E9A0>.create_report
E        +      where <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000021873D4E9A0> = <pyaedt.circuit.Circuit object at 0x00000218521BFD60>.post
E        +        where <pyaedt.circuit.Circuit object at 0x00000218521BFD60> = <_unittest.test_21_Circuit.TestClass object at 0x0000021853E4EB20>.aedtapp
E        +    and   '' = <pyaedt.circuit.Circuit object at 0x00000218521BFD60>.nominal_adaptive
E        +      where <pyaedt.circuit.Circuit object at 0x00000218521BFD60> = <_unittest.test_21_Circuit.TestClass object at 0x0000021853E4EB20>.aedtapp
E        +    and   60 = <pyaedt.modeler.circuits.object3dcircuit.CircuitComponent object at 0x0000021873D55C70>.id

_unittest\test_21_Circuit.py:107: AssertionError
---------------------------------------------------------------------------------- Captured stdout setup ----------------------------------------------------------------------------------- 
pyaedt INFO: using existing logger.
pyaedt INFO: Launching PyAEDT outside AEDT with CPython and PythonNET.
pyaedt INFO: AEDT installation Path C:\Program Files\AnsysEM\v231\Win64.
pyaedt INFO: Launching AEDT with module PythonNET.
pyaedt INFO: Ansoft.ElectronicsDesktop.2023.1 Started with process ID 4396.
pyaedt INFO: pyaedt v0.7.dev0
pyaedt INFO: Python version 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)]
pyaedt INFO: AEDT file load (threaded) time: 0.16614747047424316
pyaedt INFO: Project Galileo_t21_231 has been opened.
pyaedt INFO: No consistent unique design is present. Inserting a new design.
pyaedt INFO: Added design 'Circuit Design_G5R' of type Circuit Design.
pyaedt INFO: Aedt Objects initialized
pyaedt INFO: Variable Manager initialized
pyaedt INFO: Design Loaded
pyaedt INFO: Successfully loaded project materials !
pyaedt INFO: Materials Loaded
pyaedt INFO: AEDT file load (threaded) time: 0.052933454513549805
pyaedt INFO: Project differential_pairs_231 has been opened.
pyaedt INFO: Active Design set to Circuit1
pyaedt INFO: Aedt Objects initialized
pyaedt INFO: Variable Manager initialized
pyaedt INFO: Design Loaded
pyaedt INFO: Successfully loaded project materials !
pyaedt INFO: Materials Loaded
------------------------------------------------------------------------------------ Captured log setup ------------------------------------------------------------------------------------ 
INFO     Global:aedt_logger.py:473 using existing logger.
INFO     Global:aedt_logger.py:473 Launching PyAEDT outside AEDT with CPython and PythonNET.
INFO     Global:aedt_logger.py:473 AEDT installation Path C:\Program Files\AnsysEM\v231\Win64.
INFO     Global:aedt_logger.py:473 Launching AEDT with module PythonNET.
INFO     Global:aedt_logger.py:473 Ansoft.ElectronicsDesktop.2023.1 Started with process ID 4396.
INFO     Global:aedt_logger.py:473 pyaedt v0.7.dev0
INFO     Global:aedt_logger.py:473 Python version 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)]
INFO     Global:aedt_logger.py:473 AEDT file load (threaded) time: 0.16614747047424316
INFO     Global:aedt_logger.py:473 Project Galileo_t21_231 has been opened.
INFO     Global:aedt_logger.py:473 No consistent unique design is present. Inserting a new design.
INFO     Global:aedt_logger.py:473 Added design 'Circuit Design_G5R' of type Circuit Design.
INFO     Global:aedt_logger.py:473 Aedt Objects initialized
INFO     Global:aedt_logger.py:473 Variable Manager initialized
INFO     Global:aedt_logger.py:473 Design Loaded
INFO     Global:aedt_logger.py:473 Successfully loaded project materials !
INFO     Global:aedt_logger.py:473 Materials Loaded
INFO     Global:aedt_logger.py:473 AEDT file load (threaded) time: 0.052933454513549805
INFO     Global:aedt_logger.py:473 Project differential_pairs_231 has been opened.
INFO     Global:aedt_logger.py:473 Active Design set to Circuit1
INFO     Global:aedt_logger.py:473 Aedt Objects initialized
INFO     Global:aedt_logger.py:473 Variable Manager initialized
INFO     Global:aedt_logger.py:473 Design Loaded
INFO     Global:aedt_logger.py:473 Successfully loaded project materials !
INFO     Global:aedt_logger.py:473 Materials Loaded
----------------------------------------------------------------------------------- Captured stdout call ----------------------------------------------------------------------------------- 
pyaedt ERROR: Setup doesn't exist in this design.
------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------- 
ERROR    Global:aedt_logger.py:477 Setup doesn't exist in this design.
____________________________________________________________________________ TestClass.test_07a_push_excitation ____________________________________________________________________________ 

self = <_unittest.test_21_Circuit.TestClass object at 0x0000021853E24E80>

    def test_07a_push_excitation(self):
>       assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=False)

_unittest\test_21_Circuit.py:126:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  
pyaedt\generic\general_methods.py:296: in wrapper
    result = user_function(*args, **kwargs)
pyaedt\circuit.py:1066: in push_excitations
    self.modeler.oeditor.PushExcitations(instance_name, arg)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

self = <COMObject SetActiveEditor>, refDes = 'U1', options = ['NAME:options', 'CalcThevenin:=', False, 'Sol:=', 'LNA']

>   ???
E   pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147467259), None)

<COMObject SetActiveEditor>:2: com_error
--------------------------------------------------------------------------------- Captured stdout teardown --------------------------------------------------------------------------------- 
pyaedt INFO: Project files removed from handlers.
---------------------------------------------------------------------------------- Captured log teardown ----------------------------------------------------------------------------------- 
INFO     Global:aedt_logger.py:473 Project files removed from handlers.
================================================================================= short test summary info ================================================================================== 
FAILED _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component - AssertionError: assert False
FAILED _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation - pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147467259), None)
==================================================================================== 2 failed in 15.02s ==================================================================================== 
```

### Steps To Reproduce

1. Checkout latest main
2. Run: `pytest _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component  _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation`

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.9

### Installed packages

aiohttp==3.8.3
aiosignal==1.3.1
appdirs==1.4.4
asttokens==2.2.1
async-timeout==4.0.2
attrs==22.1.0
backcall==0.2.0
certifi==2022.9.24
cffi==1.15.1
cfgv==3.3.1
charset-normalizer==2.1.1
click==8.1.3
click-plugins==1.1.1
cligj==0.7.2
clr-loader==0.2.4
codecov==2.1.12
colorama==0.4.6
contourpy==1.0.6
coverage==7.1.0
cycler==0.11.0
decorator==5.1.1
distlib==0.3.6
et-xmlfile==1.1.0
exceptiongroup==1.0.1
execnet==1.9.0
executing==1.2.0
filelock==3.9.0
Fiona==1.9.0
fonttools==4.38.0
frozenlist==1.3.3
geopandas==0.12.2
identify==2.5.17
idna==3.4
imageio==2.22.4
iniconfig==1.1.1
ipython==8.9.0
jedi==0.18.2
joblib==1.2.0
kiwisolver==1.4.4
matplotlib==3.6.3
matplotlib-inline==0.1.6
multidict==6.0.2
munch==2.5.0
networkx==3.0
nodeenv==1.7.0
numpy==1.24.2
openpyxl==3.1.0
osmnx==1.3.0
packaging==21.3
pandas==1.5.3
parso==0.8.3
pickleshare==0.7.5
Pillow==9.3.0
platformdirs==3.0.0
pluggy==1.0.0
plumbum==1.8.0
pooch==1.6.0
pre-commit==3.0.4
prompt-toolkit==3.0.36
psutil==5.9.4
pure-eval==0.2.2
-e git+https://github.com/pyansys/pyaedt.git@d1830e889934269ff70b0c043b2853ada3b4b84c#egg=pyaedt
pycparser==2.21
Pygments==2.14.0
pyparsing==3.0.9
pyproj==3.4.1
pytest==7.2.1
pytest-cov==4.0.0
pytest-xdist==3.2.0
python-dateutil==2.8.2
pythonnet==3.0.1
pytz==2022.6
pyvista==0.38.1
pywin32==305
PyYAML==6.0
requests==2.28.1
rpyc==5.3.0
scikit-learn==1.2.1
scipy==1.10.0
scooby==0.7.0
shapely==2.0.1
six==1.16.0
SRTM.py==0.3.7
stack-data==0.6.2
threadpoolctl==3.1.0
tomli==2.0.1
traitlets==5.9.0
urllib3==1.26.12
utm==0.7.0
virtualenv==20.19.0
vtk==9.2.2
wcwidth==0.2.6
wslink==1.9.1
yarl==1.8.1

@gmalinve the branch is based on d1830e889934269ff70b0c043b2853ada3b4b84c which is the commit from main just prior to the current one committed a few minutes ago.  These two tests are failing for me in 2022 R2 at least even with the latest main, however:

```
(venv) PS C:\git\Ansys\pyaedt> pytest _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation
=================================================================================== test session starts ====================================================================================
platform win32 -- Python 3.9.12, pytest-7.2.1, pluggy-1.0.0
rootdir: C:\git\Ansys\pyaedt, configfile: pytest.ini
plugins: cov-4.0.0, xdist-3.2.0
collected 2 items

_unittest\test_21_Circuit.py FF                                                                                                                                                       [100%]

========================================================================================= FAILURES ========================================================================================= 
________________________________________________________________________ TestClass.test_06b_add_3dlayout_component _________________________________________________________________________ 

self = <_unittest.test_21_Circuit.TestClass object at 0x0000029CA95BFAC0>

    def test_06b_add_3dlayout_component(self):
        myedb = self.aedtapp.modeler.schematic.add_subcircuit_3dlayout("Galileo_G87173_204")
        assert type(myedb.id) is int
        ports = myedb.pins
        tx = ports
        rx = ports
        insertions = ["dB(S({},{}))".format(i.name, j.name) for i, j in zip(tx, rx)]
>       assert self.aedtapp.post.create_report(
            insertions,
            self.aedtapp.nominal_adaptive,
            plotname="Insertion Losses",
            plot_type="Rectangular Plot",
            report_category="Standard",
            subdesign_id=myedb.id,
        )
E       AssertionError: assert False
E        +  where False = <bound method PostProcessorCommon.create_report of <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000029CC96A1FA0>>(['dB(S(J3B2.3.USBH2_DP_CH,J3B2.3.USBH2_DP_CH))', 'dB(S(J3B2.2.USBH2_DN_CH,J3B2.2.USBH2_DN_CH))', 'dB(S(J2B3-2,J2B3-2))'], '', plotname='Insertion Losses', plot_type='Rectangular Plot', report_category='Standard', subdesign_id=60)
E        +    where <bound method PostProcessorCommon.create_report of <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000029CC96A1FA0>> = <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000029CC96A1FA0>.create_report
E        +      where <pyaedt.modules.PostProcessor.CircuitPostProcessor object at 0x0000029CC96A1FA0> = <pyaedt.circuit.Circuit object at 0x0000029CA792CF70>.post
E        +        where <pyaedt.circuit.Circuit object at 0x0000029CA792CF70> = <_unittest.test_21_Circuit.TestClass object at 0x0000029CA95BFAC0>.aedtapp
E        +    and   '' = <pyaedt.circuit.Circuit object at 0x0000029CA792CF70>.nominal_adaptive
E        +      where <pyaedt.circuit.Circuit object at 0x0000029CA792CF70> = <_unittest.test_21_Circuit.TestClass object at 0x0000029CA95BFAC0>.aedtapp
E        +    and   60 = <pyaedt.modeler.circuits.object3dcircuit.CircuitComponent object at 0x0000029CC937C4C0>.id

_unittest\test_21_Circuit.py:107: AssertionError
---------------------------------------------------------------------------------- Captured stdout setup ----------------------------------------------------------------------------------- 
pyaedt INFO: using existing logger.
pyaedt INFO: Launching PyAEDT outside AEDT with CPython and PythonNET.
pyaedt INFO: AEDT installation Path C:\Program Files\AnsysEM\v222\Win64.
pyaedt INFO: Launching AEDT with module PythonNET.
pyaedt INFO: Ansoft.ElectronicsDesktop.2022.2 Started with process ID 28768.
pyaedt INFO: pyaedt v0.7.dev0
pyaedt INFO: Python version 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)]
pyaedt INFO: AEDT file load (threaded) time: 0.16956424713134766
pyaedt INFO: Project Galileo_t21 has been opened.
pyaedt INFO: No consistent unique design is present. Inserting a new design.
pyaedt INFO: Added design 'Circuit Design_YHA' of type Circuit Design.
pyaedt INFO: Aedt Objects initialized
pyaedt INFO: Variable Manager initialized
pyaedt INFO: Design Loaded
pyaedt INFO: Successfully loaded project materials !
pyaedt INFO: Materials Loaded
pyaedt INFO: AEDT file load (threaded) time: 0.04819369316101074
pyaedt INFO: Project differential_pairs has been opened.
pyaedt INFO: Active Design set to Circuit1
pyaedt INFO: Aedt Objects initialized
pyaedt INFO: Variable Manager initialized
pyaedt INFO: Design Loaded
pyaedt INFO: Successfully loaded project materials !
pyaedt INFO: Materials Loaded
------------------------------------------------------------------------------------ Captured log setup ------------------------------------------------------------------------------------ 
INFO     Global:aedt_logger.py:473 using existing logger.
INFO     Global:aedt_logger.py:473 Launching PyAEDT outside AEDT with CPython and PythonNET.
INFO     Global:aedt_logger.py:473 AEDT installation Path C:\Program Files\AnsysEM\v222\Win64.
INFO     Global:aedt_logger.py:473 Launching AEDT with module PythonNET.
INFO     Global:aedt_logger.py:473 Ansoft.ElectronicsDesktop.2022.2 Started with process ID 28768.
INFO     Global:aedt_logger.py:473 pyaedt v0.7.dev0
INFO     Global:aedt_logger.py:473 Python version 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)]
INFO     Global:aedt_logger.py:473 AEDT file load (threaded) time: 0.16956424713134766
INFO     Global:aedt_logger.py:473 Project Galileo_t21 has been opened.
INFO     Global:aedt_logger.py:473 No consistent unique design is present. Inserting a new design.
INFO     Global:aedt_logger.py:473 Added design 'Circuit Design_YHA' of type Circuit Design.
INFO     Global:aedt_logger.py:473 Aedt Objects initialized
INFO     Global:aedt_logger.py:473 Variable Manager initialized
INFO     Global:aedt_logger.py:473 Design Loaded
INFO     Global:aedt_logger.py:473 Successfully loaded project materials !
INFO     Global:aedt_logger.py:473 Materials Loaded
INFO     Global:aedt_logger.py:473 AEDT file load (threaded) time: 0.04819369316101074
INFO     Global:aedt_logger.py:473 Project differential_pairs has been opened.
INFO     Global:aedt_logger.py:473 Active Design set to Circuit1
INFO     Global:aedt_logger.py:473 Aedt Objects initialized
INFO     Global:aedt_logger.py:473 Variable Manager initialized
INFO     Global:aedt_logger.py:473 Design Loaded
INFO     Global:aedt_logger.py:473 Successfully loaded project materials !
INFO     Global:aedt_logger.py:473 Materials Loaded
----------------------------------------------------------------------------------- Captured stdout call ----------------------------------------------------------------------------------- 
pyaedt ERROR: Setup doesn't exist in this design.
------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------- 
ERROR    Global:aedt_logger.py:477 Setup doesn't exist in this design.
____________________________________________________________________________ TestClass.test_07a_push_excitation ____________________________________________________________________________ 

self = <_unittest.test_21_Circuit.TestClass object at 0x0000029CA9595E80>

    def test_07a_push_excitation(self):
>       assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=False)

_unittest\test_21_Circuit.py:126:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  
pyaedt\generic\general_methods.py:296: in wrapper
    result = user_function(*args, **kwargs)
pyaedt\circuit.py:1066: in push_excitations
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

self = <COMObject SetActiveEditor>, refDes = 'U1', options = ['NAME:options', 'CalcThevenin:=', False, 'Sol:=', 'LNA']

>   ???
E   pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147467259), None)

<COMObject SetActiveEditor>:2: com_error
--------------------------------------------------------------------------------- Captured stdout teardown --------------------------------------------------------------------------------- 
pyaedt INFO: Project files removed from handlers.
---------------------------------------------------------------------------------- Captured log teardown ----------------------------------------------------------------------------------- 
INFO     Global:aedt_logger.py:473 Project files removed from handlers.
================================================================================= short test summary info ================================================================================== 
FAILED _unittest/test_21_Circuit.py::TestClass::test_06b_add_3dlayout_component - AssertionError: assert False
FAILED _unittest/test_21_Circuit.py::TestClass::test_07a_push_excitation - pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147467259), None)
==================================================================================== 2 failed in 28.84s ==================================================================================== 
(venv) PS C:\git\Ansys\pyaedt> git log -1
commit 066e2ce7103cab9c405eb4b662ddcb658ccd5f67 (HEAD -> main, origin/main, origin/HEAD)
Author: JDufrenneAnsys <125255495+JDufrenneAnsys@users.noreply.github.com>
Date:   Tue Feb 14 10:10:41 2023 +0000

    2454 : Plot brd outline (#2455)

    * 2454 : Plot brd outline

    - Allows to plot a layer not related to a net
    - If layer name is "Outline" plot it as a contour

    * [pre-commit.ci] auto fixes from pre-commit.com hooks

    for more information, see https://pre-commit.ci

    ---------
```

--- a/_unittest/test_21_Circuit.py
+++ b/_unittest/test_21_Circuit.py
@@ -98,7 +98,7 @@ class TestClass(BasisTest, object):
         assert LNA_setup.name == "LNA"
 
     def test_06b_add_3dlayout_component(self):
-        setup = self.aedtapp.create_setup("LNA")
+        setup = self.aedtapp.create_setup("test_06b_LNA")
         setup.add_sweep_step(start_point=0, end_point=5, step_size=0.01)
         myedb = self.aedtapp.modeler.schematic.add_subcircuit_3dlayout("Galileo_G87173_204")
         assert type(myedb.id) is int
@@ -125,10 +125,11 @@ class TestClass(BasisTest, object):
         assert type(my_model) is int
 
     def test_07a_push_excitation(self):
-        setup = self.aedtapp.create_setup("LNA")
+        setup_name = "test_07a_LNA"
+        setup = self.aedtapp.create_setup(setup_name)
         setup.add_sweep_step(start_point=0, end_point=5, step_size=0.01)
-        assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=False)
-        assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=True)
+        assert self.aedtapp.push_excitations(instance_name="U1", setup_name=setup_name, thevenin_calculation=False)
+        assert self.aedtapp.push_excitations(instance_name="U1", setup_name=setup_name, thevenin_calculation=True)
 
     def test_08_import_mentor_netlist(self):
         self.aedtapp.insert_design("MentorSchematicImport")

--- a/_unittest/test_21_Circuit.py
+++ b/_unittest/test_21_Circuit.py
@@ -98,6 +98,8 @@ class TestClass(BasisTest, object):
         assert LNA_setup.name == "LNA"
 
     def test_06b_add_3dlayout_component(self):
+        setup = self.aedtapp.create_setup("LNA")
+        setup.add_sweep_step(start_point=0, end_point=5, step_size=0.01)
         myedb = self.aedtapp.modeler.schematic.add_subcircuit_3dlayout("Galileo_G87173_204")
         assert type(myedb.id) is int
         ports = myedb.pins
@@ -123,6 +125,8 @@ class TestClass(BasisTest, object):
         assert type(my_model) is int
 
     def test_07a_push_excitation(self):
+        setup = self.aedtapp.create_setup("LNA")
+        setup.add_sweep_step(start_point=0, end_point=5, step_size=0.01)
         assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=False)
         assert self.aedtapp.push_excitations(instance_name="U1", setup_name="LNA", thevenin_calculation=True)
 

--- a/pyaedt/generic/general_methods.py
+++ b/pyaedt/generic/general_methods.py
@@ -922,14 +922,22 @@ def com_active_sessions(version=None, student_version=False, non_graphical=False
         keys = ["ansysedtsv.exe"]
     else:
         keys = ["ansysedt.exe"]
+    long_version = None
+    if len(version) > 6:
+        version = version[-6:]
     if version and "." in version:
+        long_version = version
         version = version[-4:].replace(".", "")
-    if version < "222":
+    if version < "221":
         version = version[:2] + "." + version[2]
+        long_version = "20{}".format(version)
     sessions = []
     for p in psutil.process_iter():
         try:
             if p.name() in keys:
+                if long_version and _check_installed_version(os.path.dirname(p.exe()), long_version):
+                    sessions.append(p.pid)
+                    continue
                 cmd = p.cmdline()
                 if non_graphical and "-ng" in cmd or not non_graphical:
                     if not version or (version and version in cmd[0]):
@@ -1391,6 +1399,34 @@ def _dim_arg(value, units):
         return str(val) + units
     except:
         return value
+
+
+@pyaedt_function_handler()
+def _check_installed_version(install_path, long_version):
+    """Check installation folder to determine if it is for specified Ansys EM version.
+
+    Parameters
+    ----------
+    install_path: str
+        Installation folder to check.  For example, ``"C:\\Program Files\\AnsysEM\\v231\\Win64"``.
+    long_version: str
+        Long form of version number.  For example, ``"2023.1"``.
+
+    Returns
+    -------
+    bool
+
+    """
+    product_list_path = os.path.join(install_path, "config", "ProductList.txt")
+    if os.path.isfile(product_list_path):
+        try:
+            with open(product_list_path, "r") as f:
+                install_version = f.readline().strip()[-6:]
+                if install_version == long_version:
+                    return True
+        except:
+            pass
+    return False
 
 
 class Settings(object):

